test/unit/saveArticles.test.ts

```diff
@@ -261,3 +260,16 @@ test('treat multiple subtitles in one video', async(t) => {
         'Video multiple subtitles rewriting matches');
     t.equals(contentRes.subtitles.length, 4, 'All subtitles are found for this video');
 });
+
+test('Desktop article Rendering Test' , async(t) => {
+    const DELETED_ARTICLE_FAIL_MESSAGE = 'Throwing error if article is deleted';
```

Comment:
You should no redeclare the error message (which btw is not the same as in the code). Yo shoukd test that the exception is gneraged properly.

Reply:
@kelson42 this is not the message used for error, it is the message shown in the terminal when the integration test fails or passes, I had to put it in 2 places one if `renderDesktopArticle()` didn't throw any error it should fail, and once in the catch block if the error is different.

Reply:
Ok, but you should just success if proper exception... without having to create a fake exception.